multicluster/controllers/multicluster/common/common_area.go

```diff
@@ -0,0 +1,20 @@
+package common
+
+import (
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// Access to common area of a cluster
+// Inbound mode:
```

Comment:
Again suggest not to mention inbound mode, which is confusing to readers without contexts. If you really want to talk about it, at least add comments somewhere to describe what is inbound mode.